src/style.css

```diff
@@ -2,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). It appears you are adding margin-right to the `.stars__star:not(:last-child)` selector. Ensure that you are consistent with the direction of your margins throughout your project to avoid potential margin collapse issues.